### PR TITLE
[7.3] IsNullFixer - fix trailing comma

### PR DESCRIPTION
**Pull request by guilliamxavier**
Includes and fixes #4226

I also had to include #4229 first

### CODE REVIEW

File: tests/Fixer/LanguageConstruct/IsNullFixerTest.php [hunk 1]
```diff
@@ -224,7 +224,7 @@ public function provideFixCases()
                 '<?php if ((is_null($u) or $v) and ($w || is_null($x)) xor (!is_null($y) and $z)) echo "foo"; ?>',
             ],
 
-            // edge cases: contains dangerous constructs, wrapped into parentheses
+            // edge cases: $isContainingDangerousConstructs, $wrapIntoParentheses
```
Comment (guilliamxavier) [inline tests/Fixer/LanguageConstruct/IsNullFixerTest.php]: @keradus I guess you want to revert this back, and instead make the opposite change on line 315

Comment (keradus) [reply tests/Fixer/LanguageConstruct/IsNullFixerTest.php]: I spot from where you took those vars, I didn't saw it in prev PR

**Comment by keradus** [comment]
Thank you @guilliamxavier.
